shell/platform/glfw/platform_handler.cc

```diff
@@ -86,8 +86,8 @@ void PlatformHandler::HandleMethodCall(
     glfwSetClipboardString(window_, itr->value.GetString());
     result->Success();
   } else if (method.compare(kSystemNavigatorPopMethod) == 0) {
-    exit(EXIT_SUCCESS);
     result->Success();
+    exit(EXIT_SUCCESS);
```

Comment:
Isn't this a bug? The result is never going to have a change to sent over the channel for the other side to respond.

Reply:
Initially I figured I'd leave the `result->Success` there for consistency.  But it does not really serve a purpose - the `result` will not be sent to the client after the process exits.

Removed the `result->Success` call`.